CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Cromwell Change Log
 
+## 30 Release Notes
+
+### Other changes
+
+* **Database**  
+If desired, the metadata may be optionally stored in a separate SQL database than the of the operations data. See the
+[README](https://github.com/broadinstitute/cromwell#database) or the `database` section in
+[cromwell.examples.conf](https://github.com/broadinstitute/cromwell/blob/develop/cromwell.examples.conf) for more
+information. Cromwell will not move any existing data automatically. When switching connection information for an
```

Comment:
I suspect that the people I have in mind will get even more excited at that but I'm willing to give it a trial balloon here :)